README.md

```diff
@@ -56,7 +56,7 @@ Main features:
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
->  click here to see the [Live Demo](https://shella12.github.io/pollutionCheck/)
+>  click here to see the [Live Demo](https://shella12.github.io/air-quality-index/)
```

Comment:
- [ ] Good work making the video presentation about your project, kindly make sure to take advantage of the 5 min max video length and include a highlight about a piece of the code you find interesting or you're proud of making.

Reply:
- [ ] [OPRIONAL] Great presentation @shella12 👍🏽 However, I would advise you to enable your camera so the viewers can see your face. This will make the presentation look more professional.